explain/setup/heroku/_01_public.mdx

```diff
@@ -0,0 +1,29 @@
+# Check if auto_explain already enabled
+
+First, to check if `auto_explain` is already enabled on your system, run the
+following query with `heroku pg:psql`:
+
+```
+SHOW session_preload_libraries;
+```
+
+If `auto_explain` is already there, it is already enabled.
```

Comment:
```suggestion
If `auto_explain` is already there, it is enabled.
```
It feels weird to repeat "already" here.

Reply:
good point, I'll also update the other place too 👍